cfgov/unprocessed/css/molecules/image-text-50-50.less

```diff
@@ -82,7 +82,6 @@
 */
 
 .m-image-text-50-50 {
-    padding-bottom: unit( @grid_gutter-width * 2 / @base-font-size-px, em );
```

Comment:
Yeah, you have to be able to add as many as you want to the group because of the `h2` heading at the top of the group. You wouldn't want to break up the doc outline to fit a layout restriction.